cocli/README.md

```diff
@@ -156,21 +156,128 @@ $ cocli comid display -f m1.cbor \
                     -d yet-another-comid-folder/
 ```
 
+## CoTSs manipulation
+
+The `cots` subcommand allows you to create, display and validate CoTSs.
+
+### Create
+
+Use the `cots create` subcommand to create a CBOR-encoded CoTS:
```

Comment:
Please refer clearly what are the supplied inputs, like as done in CoMID above. From the below description it is not clear is c1 a template and what is tas_dir?